### PR TITLE
[FIX] knowledge: make macros more robust

### PR DESCRIPTION
**Pull request by abd-msyukyu-odoo**
To avoid relying on translated attributes, add classes on specific elements that
Knowledge macros can interact with.

task-3410128

### CODE REVIEW

**Review by seb-odoo — APPROVED** [review]

File: addons/mail/static/src/composer/composer.xml [hunk 1]
```diff
@@ -76,11 +76,11 @@
                             <button class="btn border-0 fa fa-smile-o rounded-pill" aria-label="Emojis" t-on-click="onClickAddEmoji" t-ref="emoji-picker"/>
                             <FileUploader t-if="allowUpload" multiUpload="true" onUploaded="(data) => { attachmentUploader.uploadData(data) }">
                                 <t t-set-slot="toggler">
-                                    <button t-att-disabled="!state.active" class="btn fa fa-paperclip border-0 rounded-pill" title="Attach files" aria-label="Attach files" type="button" t-on-click="onClickAddAttachment"/>
+                                    <button t-att-disabled="!state.active" class="o-mail-Composer-attachFiles btn fa fa-paperclip border-0 rounded-pill" title="Attach files" aria-label="Attach files" type="button" t-on-click="onClickAddAttachment"/>
```
Comment (awa-odoo) [inline addons/mail/static/src/composer/composer.xml:79]: Maybe "o-mail-Composer-action-attachFiles" and "o-mail-Composer-action-fullComposer" as those are sub-actions of the "o-mail-Composer-actions" node?

**Review by awa-odoo — COMMENTED** [review]
Hello @abd-msyukyu-odoo 

Only one small comment.

A question about your commit message however, you say that these attributes are not translated currently, but I though "title" attributes were translated, is that not the case?

Thanks & cheers!

File: addons/mail/static/src/composer/composer.xml (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/composer/composer.xml:79]: We follow the BEM nomenclature `o-moduleName-ComponentName-partName`, so keeping just `attachFiles` is good.
If the "action" part is useful e.g. to style all actions in CSS, better make another classname e.g. `o-mail-Composer-action`. But in general the less classnames the better :)

**Comment by abd-msyukyu-odoo** [comment]
@awa-odoo yup, title is indeed translated, along with a bunch of other attributes specified in `translate.py` thanks for pointing it out as this is good to know ;p

**Pull request closed without merging**